File: okj/szoftverfejleszto/jackie_stewart/java/Verseny.java
public class Verseny {

    public final int ev;
    public final int indulasokSzama;
    public final int nyeresekSzama;
    public final int dobogosokSzama;
    public final int elsoHelyrolIndulasokSzama;
    public final int leggyorsabbKorokSzama;

    public Verseny(String line) {
        var split = line.split("\t");

        ev = Integer.parseInt(split[0]);
        indulasokSzama = Integer.parseInt(split[1]);
        nyeresekSzama = Integer.parseInt(split[2]);
        dobogosokSzama = Integer.parseInt(split[3]);
        elsoHelyrolIndulasokSzama = Integer.parseInt(split[4]);
        leggyorsabbKorokSzama = Integer.parseInt(split[5]);
    }
}

File: okj/szoftverfejleszto/jackie_stewart/c#/Verseny.cs
public class Verseny {

    public readonly int ev;
    public readonly int indulasokSzama;
    public readonly int nyeresekSzama;
    public readonly int dobogosokSzama;
    public readonly int elsoHelyrolIndulasokSzama;
    public readonly int leggyorsabbKorokSzama;

    public Verseny(string line) {
        var split = line.Split('\t');

        ev = int.Parse(split[0]);
        indulasokSzama = int.Parse(split[1]);
        nyeresekSzama = int.Parse(split[2]);
        dobogosokSzama = int.Parse(split[3]);
        elsoHelyrolIndulasokSzama = int.Parse(split[4]);
        leggyorsabbKorokSzama = int.Parse(split[5]);
    }
}